installer/install.iss

```diff
@@ -87,7 +87,7 @@ WizardImageBackColor=clWhite
 WizardImageStretch=no
 WizardImageFile={#SourcePath}\git.bmp
 WizardSmallImageFile={#SourcePath}\gitsmall.bmp
-MinVersion=0,5.01sp3
+MinVersion=6.0
```

Comment:
Bumped the MinVersion to 6.0 (Vista)

Reply:
I guess that means we could drop our own check wether we're running on vista or later.

Reply:
It needs to stay there, otherwise InnoSetup [will default to](https://jrsoftware.org/isdl.php) `6.1sp1`:

> **Change in default behavior**: Starting with Inno Setup 6.1 the [Setup] section directive MinVersion defaults to 6.1sp1, so **by default Setup will not run on Windows Vista or on versions of Windows 7 and Windows Server 2008 R2 which have not been updated**. Setting MinVersion to 6.0 to allow Setup to run on Windows Vista is supported but not recommended: Windows Vista doesn't support some of Setup's security measures against potential DLL preloading attacks so these have to be removed by the compiler if MinVersion is below 6.1 making your installer less secure on all versions of Windows.

Reply:
No, i meant since InnoSetups requirement and this `MinVersion=6.0` check are in place, we don't need this check anymore: https://github.com/git-for-windows/build-extra/blob/dbd9c2cf7dc82558922d97a0261e7fab150979e6/installer/install.iss#L1000-L1006